homeassistant/components/lg_thinq/sensor.py

```diff
@@ -0,0 +1,590 @@
+    ),
+    ThinQProperty.TOTAL_POLLUTION: SensorEntityDescription(
+        key=ThinQProperty.TOTAL_POLLUTION,
+        translation_key=ThinQProperty.TOTAL_POLLUTION,
+    ),
+    ThinQProperty.MONITORING_ENABLED: SensorEntityDescription(
+        key=ThinQProperty.MONITORING_ENABLED,
+        translation_key=ThinQProperty.MONITORING_ENABLED,
+    ),
+    ThinQProperty.TEMPERATURE: SensorEntityDescription(
+        key=ThinQProperty.TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key=ThinQProperty.TEMPERATURE,
+    ),
+}
+BATTERY_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.BATTERY_LEVEL: SensorEntityDescription(
+        key=ThinQProperty.BATTERY_LEVEL,
+        translation_key=ThinQProperty.BATTERY_LEVEL,
+    ),
+    ThinQProperty.BATTERY_PERCENT: SensorEntityDescription(
+        key=ThinQProperty.BATTERY_PERCENT,
+        native_unit_of_measurement=PERCENTAGE,
+        translation_key=ThinQProperty.BATTERY_PERCENT,
+    ),
+}
+DETERGENT_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.DETERGENT_SETTING: SensorEntityDescription(
+        key=ThinQProperty.DETERGENT_SETTING,
+        translation_key=ThinQProperty.DETERGENT_SETTING,
+    ),
+}
+DISH_WASHING_COURSE_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.CURRENT_DISH_WASHING_COURSE: SensorEntityDescription(
+        key=ThinQProperty.CURRENT_DISH_WASHING_COURSE,
+        translation_key=ThinQProperty.CURRENT_DISH_WASHING_COURSE,
+    )
+}
+DOOR_STATUS_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.DOOR_STATE: SensorEntityDescription(
+        key=ThinQProperty.DOOR_STATE,
+        translation_key=ThinQProperty.DOOR_STATE,
+    ),
+}
+FILTER_INFO_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.FILTER_LIFETIME: SensorEntityDescription(
+        key=ThinQProperty.FILTER_LIFETIME,
+        native_unit_of_measurement=UnitOfTime.HOURS,
+        translation_key=ThinQProperty.FILTER_LIFETIME,
+    ),
+}
+HUMIDITY_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.CURRENT_HUMIDITY: SensorEntityDescription(
+        key=ThinQProperty.CURRENT_HUMIDITY,
+        device_class=SensorDeviceClass.HUMIDITY,
+        native_unit_of_measurement=PERCENTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key=ThinQProperty.CURRENT_HUMIDITY,
+    )
+}
+JOB_MODE_SENSOR_DESC: dict[ThinQProperty, SensorEntityDescription] = {
+    ThinQProperty.CURRENT_JOB_MODE: SensorEntityDescription(
+        key=ThinQProperty.CURRENT_JOB_MODE,
+        translation_key=ThinQProperty.CURRENT_JOB_MODE,
+    ),
+    ThinQPropertyEx.CURRENT_JOB_MODE_STICK_CLEANER: SensorEntityDescription(
+        key=ThinQProperty.CURRENT_JOB_MODE,
+        translation_key=ThinQPropertyEx.CURRENT_JOB_MODE_STICK_CLEANER,
+    ),
```

Comment:
What does this do?

Reply:
It is a job mode for Stick cleaner.


R | OFF | OFF
-- | -- | --
R | NORMAL | NORMAL
R | HIGH | HIGH
R | TURBO | TURBO
R | MOP | MOP
R | AUTO | AUTO